app/views/admin/consultations/_form.html.erb

```diff
@@ -1,7 +1,6 @@
 <div class="format-advice">
-  <p class="govuk-body"><strong>Use this format for:</strong> Consultations (officially “documents requiring collective agreement across government”, including calls for evidence).</p>
-
-  <p class="govuk-body">Check you have read the <%= link_to "consultation principles", "https://www.gov.uk/government/publications/consultation-principles-guidance", class: "govuk-link" %>.</p>
+  <p class="govuk-body"><strong>Use this format for:</strong> <%= link_to "Consultations", "https://www.gov.uk/guidance/content-design/content-types", class: "govuk-link" %>following the <%= link_to "consultation principles", "https://www.gov.uk/government/publications/consultation-principles-guidance", class: "govuk-link" %>.</p>
```

Comment:
It looks like you're missing a space between the link to "Consultations" and the next words "following the".

```suggestion
  <p class="govuk-body"><strong>Use this format for:</strong> <%= link_to "Consultations", "https://www.gov.uk/guidance/content-design/content-types", class: "govuk-link" %> following the <%= link_to "consultation principles", "https://www.gov.uk/government/publications/consultation-principles-guidance", class: "govuk-link" %>.</p>
```